go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -17,72 +17,102 @@ package main
 import (
 	"context"
 	"fmt"
+	"path/filepath"
 	"strings"
 )
 
 type BazelJSONBuilder struct {
-	bazel      *Bazel
-	query      string
-	tagFilters string
-	targets    []string
+	bazel    *Bazel
+	requests []string
 }
 
 const (
 	OutputGroupDriverJSONFile = "go_pkg_driver_json_file"
 	OutputGroupStdLibJSONFile = "go_pkg_driver_stdlib_json_file"
 	OutputGroupExportFile     = "go_pkg_driver_export_file"
+	wildcardQuery             = `kind("go_library", //...)`
 )
 
-func NewBazelJSONBuilder(bazel *Bazel, query, tagFilters string, targets []string) (*BazelJSONBuilder, error) {
+func (b *BazelJSONBuilder) fileQuery(filename string) string {
+	if filepath.IsAbs(filename) {
+		fp, _ := filepath.Rel(b.bazel.WorkspaceRoot(), filename)
+		filename = fp
+	}
+	return fmt.Sprintf(`some(kind("go_library", same_pkg_direct_rdeps("%s")))`, filename)
+}
+
+func (b *BazelJSONBuilder) packageQuery(importPath string) string {
+	return fmt.Sprintf(`some(kind("go_library", attr(importpath, "%s", %s)))`, importPath, bazelQueryScope)
```

Comment:
Should be:
```suggestion
	return fmt.Sprintf(`some(kind("go_library", attr(importpath, "%s", deps(%s))))`, importPath, bazelQueryScope)
```
so we can get package information from the dependencies